pages/media/index.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="media-wrapper">
+    <div
+      ref="player"
+    />
+  </div>
+</template>
+
+<script>
+  export default {
+    layout: 'minimal',
+
+    data() {
+      return {
+        // TODO: version and use unpkg with @europeana/media-player pkg, when available
+        MEDIA_PLAYER_SRC: 'https://ec-3505-media-player.eu-de.mybluemix.net/europeana-media-player.min.js'
+      };
+    },
+
+    computed: {
+      manifest() {
+        return `https://iiif.europeana.eu/presentation/${this.id}/manifest?format=3`;
+      }
+    },
+
+    asyncData({ query }) {
+      return {
+        id: query.id
+      };
+    },
+
+    mounted() {
+      this.$nextTick(() => {
+        new EuropeanaMediaPlayer(this.$refs.player, { // eslint-disable-line no-undef
+          manifest: this.manifest
+        });
+      });
+    },
+
+    head() {
+      return {
+        title: 'Media player',
+
+        script: [
+          { src: 'https://code.jquery.com/jquery-3.4.1.min.js' },
+          { src: 'https://code.jquery.com/ui/1.12.1/jquery-ui.min.js' },
```

Comment:
jquery & jquery ui are dependencies for the media player? anyway we can avoid using them and replace with something like https://umbrellajs.com/ ?

Reply:
I can get the volume percentage displaying with a little bit of css (background-color & height), this would be an ok fix, though showing the handle too would be best

<img width="149" alt="Screenshot 2020-05-13 at 17 27 21" src="https://user-images.githubusercontent.com/4670058/81789500-09e98a80-953f-11ea-8502-834adcc9e040.png">